.github/workflows/bazel.yml

```diff
@@ -291,9 +291,31 @@ jobs:
           run: |
             cd /workspaces/magma
             bazel/scripts/remote_cache_bazelrc_setup.sh "${{ env.CACHE_KEY }}" "${{ env.REMOTE_DOWNLOAD_OPTIMIZATION }}" "${{ secrets.BAZEL_REMOTE_PASSWORD }}"
-            bazel build lte/gateway/release:sctpd_deb_pkg lte/gateway/release:magma_deb_pkg \
+            bazel run //lte/gateway/release:release_build \
               --config=production \
               --profile=Bazel_build_package_profile
+            mkdir packages
+            mv /tmp/packages/*.deb packages
+      - name: Setup JFROG CLI
+        uses: jfrog/setup-jfrog-cli@d0a59b1cdaeeb16e65b5039fc92b8507337f1559 # pin@v3
+        env:
+          JF_ENV_1: ${{ secrets.JFROG_TOKEN }}
+      - name: Set dry run environment variable
+        if: ${{ ! ( github.event_name == 'push' && github.repository_owner == 'magma' ) }}
+        run: |
+          echo "IS_DRY=--dry-run" >> $GITHUB_ENV
+      - name: Publish debian packages
+        env:
+          ARTIFACTORY_URL: https://artifactory.magmacore.org:443/artifactory/
+          DEBIAN_META_INFO: deb.distribution=focal-ci;deb.component=main;deb.architecture=amd64
+        run: |
+          jf rt u \
```

Comment:
I would use the long command "upload" instead of "u" here

Reply:
:+1: changed